components/user-avatar/src/user-avatar.js

```diff
@@ -37,6 +37,8 @@ const UserAvatar = ({
             extralarge,
         })}
         data-test={dataTest}
+        aria-label={name}
```

Comment:
I think the requirements [here](https://dhis2.atlassian.net/browse/LIBS-573) is to expose a property `aria-label` in the props and use it (maybe you can fallback to `name` if that doesn't exist but should expose the property nevertheless.

For `text-avatar`, it can remain as it is

Reply:
Added the improvements